src/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+function TodoList() {
+  return (
+    <ul>
+      {todoList.map(function (item) {
+        return (<li key={item.id}>{item.title}</li>)
+      })}
+    </ul>
+  )
+}
+
+export default TodoList;
+
+const todoList = [
```

Comment:
Great work! The page is looking good. It will be better if you move todoList array before TodoList function.